linux_os/guide/system/network/network-ipv6/configuring_ipv6/sysctl_net_ipv6_conf_all_accept_source_route/ignition/shared.yml

```diff
@@ -1,4 +1,4 @@
-# platform = multi_platform_ocp
+# platform = multi_platform_rhel,multi_platform_fedora,multi_platform_ocp
```

Comment:
Could you clarify what is the intent of making Ignition remediations applicable on RHEL and Fedora?

Reply:
From another issue "These are still needed for RHEL worker nodes. Trying to be a good upstream OKD steward and added Fedora too. The ignition scripts won't get added to the RHEL and Fedora datastreams without these lines."